src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__item-img a {
```

Comment:
It's a bad practice to add styles using a combined selector
it's better to add class to `a` tag and separate class for `img` tag and add styles using these classes selectors separately